R/pipelines.R

```diff
@@ -519,9 +519,54 @@ pipeline_survtoregr = function(method = 1, regr_learner = lrn("regr.featureless"
   gr
 }
 
+#' @name mlr_graphs_survtoclassif
+#' @title Survival to Classification Reduction Pipeline
+#' @description Wrapper around multiple [PipeOp][mlr3pipelines::PipeOp]s to help in creation
+#' of complex survival reduction methods.
+#'
+#' @param learner [LearnerClassif][mlr3::LearnerClassif]\cr
+#' Classification learner to fit the transformed [TaskClassif][mlr3::TaskClassif].
+#' `learner` must have `predict_type` of type `"prob"`.
+#' @param cut `numeric()`\cr
+#' Split points, used to partition the data into intervals.
+#' If unspecified, all unique event times will be used.
+#' @param max_time `numeric(1)`\cr
+#' If cut is unspecified, this will be the last possible event time.
+#' All event times after max_time will be administratively censored at max_time.
+#'
+#' @return [mlr3pipelines::Graph] or [mlr3pipelines::GraphLearner]
+#' @family pipelines
+#'
+#' @examples
+#' \dontrun{
+#' if (requireNamespace("mlr3pipelines", quietly = TRUE)) {
+#'   library("mlr3")
+#'   library("mlr3pipelines")
+#'
+#'   task = tsk("rats")
+#'
+#'   pipe = ppl(
+#'     "survtoclassif",
+#'     learner = lrn("classif.log_reg")
+#'   )
+#'   pipe$train(task)
+#'   pipe$predict(task)
+#'
+#' }
+#' }
+#' @export
+pipeline_survtoclassif = function(learner, cut = NULL, max_time = NULL) {
```

Comment:
please add the `graph_learner` parameter, see other pipelines:
```
 if (graph_learner) {
    gr = mlr3pipelines::GraphLearner$new(gr)
  }
```

this will make the whole pipeline behave like a learner, so easier to test as well